### DISCUSSION TITLE
Updating firmware in the crazyflie 2.x

### DISCUSSION THREAD

**Discussion by Bikerholic** [bitcraze/discussions#84] [Q&A] [answered]
Hi im tharun i have updated my crazy flie2.x firmware today the process of firmware updated has completed succesfully but after updating the crazyflie It doesnt ON and only one blue light is blinking what want to do for rectifying my problem please help me thanks in advance....

**Comment by krichardsson** [comment, marked as answer]
Hi!

This happens if the wrong flavor of firmware is flashed to a device (for instance the tag or bolt firmware on a Crazyflie), or if the NRF firmware is corrupt. Make sure the name of the firmware in the drop down contains something like "firmware-cf2".

You can find recovery instructions in the [client documentation](https://www.bitcraze.io/documentation/repository/crazyflie-clients-python/master/userguides/recovery-mode/)

**Reply by Bikerholic** [reply]
Thank you very much
